pkg/provider/identity_test.go

```diff
@@ -184,7 +187,7 @@ func TestNewServerOptions(t *testing.T) {
 	done, _, fi, _ := tempIdentity(t)
 	defer done()
 
-	tmp, err := ioutil.TempDir("", "TestNewServerOptions")
+	tmp, err := ioutil.TempDir(os.TempDir(), "")
```

Comment:
When you leave the first argument off it will default to `os.TempDir`, but see my PR how to handle temporary directories more nicely. Otherwise it's easy to forget to properly check the removing of things.

Reply:
right - i saw that in another PR,  will remember going forward - thanks!